actions/add-review-url/lib/index.js

```diff
@@ -28,7 +28,7 @@ function run() {
         const addComment = new AddReviewUrl_1.default();
         try {
             const host = core.getInput('host');
-            let headref = process.env.GITHUB_HEAD_REF || '';
+            let headref = core.getInput('child-namespace') || '';
             const comment = `You can see a private version of the changes made in this pull request here:\nhttp://${headref}.s.${host}/`;

```

Comment:
Something I noticed in the demo is that we hard-code `http` here. I don't know of any way we could easily determine whether it should be `http` or `https`, so we might need another parameter for that.

Reply:
Will add a task for this. Let's discuss and close on a design.

Reply:
Added an additional optional parameter for the custom action which defaults to http.